EQL Docs for Timeline (#554)
elastic/security-docs: docs/events/timeline-ui-overview.asciidoc

+
NOTE: Each Timeline object in the file must be represented in a single line.
Multiple Timeline objects are delimited with newlines.

[discrete]
[[filter-with-eql]]
== Filter Timeline Results with EQL

> Use the *Correlations* tab to investigate Timeline results with EQL queries. You can enter EQL queries into the free-form text box under this tab and a link to EQL documentation is available in the bottom right corner of the text box.

Reviewer: Added a link to the the EQL docs, which is also in the lower-right corner as you originally stated. But I don't think we need to point it out in the docs. 

```suggestion
Use the *Correlations* tab to investigate Timeline results with {ref}/eql.html[EQL queries]. 
```

When forming EQL queries, you can write a basic query to return a list of events and alerts. Or, you can create sequences of EQL queries to view matched, ordered events across multiple event categories. Sequence queries are useful for identifying and predicting related events. They can also provide a more complete picture of potential adversary behavior in your environment, which you can use to create or update rules and detection alerts.

> The following image shows what matched ordered events look like in the Timeline table. Events are paired into sets of two and shaded red or blue to indicate that they are matched:

Reviewer 2: A sample image for consideration:

<img width="1793" alt="correlation-tab" src="https://user-images.githubusercontent.com/4459398/111673392-8b095200-87e0-11eb-9c41-a09c2aba2ef3.png">

> The following image shows what matched ordered events look like in the Timeline table. Events are paired into sets of two and shaded red or blue to indicate that they are matched:

Reviewer 2: Consider changing

> Events are paired into sets of two and shaded red or blue to indicate that they are matched

to something like

`Events belonging to the same sequence are grouped together with alternating blue or red shading, and ordered by time within the group.`

- The ~into sets of two~ part is removed because a sequence can contain more than two events / alerts
- Alternatively, the additional suggested information, `,and ordered by time within the group`  could be conveyed as a separate statement, (without the `and` conjunction)

From the *Correlations* tab you can also perform the following:

* Specify the span of time that you want to investigate.
* Re-order the columns and choose which fields display.

> * Choose whether you want to see events, detection alerts, all data sources, or a custom data source. All data sources are displayed by default.

Reviewer 2: Consider adding the following statement / hyperlink:

> Custom data sources may include [Kibana index patterns](https://www.elastic.co/guide/en/kibana/current/index-patterns.html).

and re-ordering the enumerated list of data sources so the default, `All data sources`, is listed first, and the others follow in the same order that they appear in the UI:

`Choose whether you want to see All data sources, just events, just detection alerts, or a custom data source. Custom data sources may include Kibana index patterns. All data sources are displayed by default.`
